### PR TITLE
CLDR-14251 Modernize: remove v.jsp; add SurveyTool.java, CldrGui.js

### PR DESCRIPTION
**Pull request by btangmu**
-New SurveyTool.java with most of the Java code from v.jsp

-New CldrGui.js with most of the HTML code from v.jsp

-Removed SurveyAjax.includeEvilJavaScript; handle more such items with CldrStatus.js

-Extend statusJSON to include more info needed by front end

-More js unit tests; encapsulate parseAsMimeType in new TestCldrTest.js

-Change dojo config parseOnLoad to false, call

-In redesign/review.js, use dojo/ready instead of jquery equivalent: wait for cldrGui.run

-Skip dojo/ready where superfluous

-Use XMLHttpRequest instead of dojo/request in cldrStAjax, per vanilla-js.com

-Normalize error handlers for cldrStAjax.sendXhr: err parameter is always a string

-New CldrText.js to replace stui.str and stui.sub, no more loadStui

-New WHAT_ADMIN_PANEL so front end does not need vap for URL if logged in as admin

-Restore/revise comment about fonts in surveytool.css (removed in PR 877)

-Remove dead code; clean up

-Comments

-Some debugging code and comments that should be fixed before making a non-draft PR

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14251
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+        if (SurveyMain.isUnofficial()) {
+            out.write("<p><span class='glyphicon glyphicon-wrench'></span>"
+                + SurveyMain.getCurrev(true) + "</p>\n");
+        }
+        out.write("If you are not redirected in a minute or two, please click\n");
+        out.write("<a id='redir2' href='" + request.getContextPath() + "/survey"
+            + "'>this link</a> to try again.\n");
+        /***
+         * TODO: more code here from v.jsp...
+        if (request.getParameter("_STARTINGUP") == null) {
```
Comment (btangmu) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java:118]: This part isn't done yet

File: tools/cldr-apps/src/main/webapp/surveyTool/nls/stui.js [hunk 2]
```diff
@@ -206,7 +206,6 @@ define({
 		StatusAction_FORBID_PERMANENT_WITHOUT_FORUM: "A forum entry is required to make a Permanent vote.",
```
Comment (btangmu) [inline tools/cldr-apps/src/main/webapp/surveyTool/nls/stui.js:206]: This file can now be removed. It's replaced by CldrText.js

File: tools/cldr-apps/src/main/webapp/js/survey.js [hunk 3]
```diff
@@ -4360,5 +4212,4 @@ function showUserActivity(list, tableRef) {
 					});
 				});
 			});
-	});
 }
```
Comment (btangmu) [inline tools/cldr-apps/src/main/webapp/js/survey.js:4215]: Indentation is very screwed up -- I left it this way so diff is easier -- but let's format all the js files soon, maybe with "npx prettier" default options
